src/client_side.cc

```diff
@@ -2647,7 +2647,7 @@ Security::ContextPointer
 ConnStateData::getTlsContextFromCache(const SBuf &cacheKey, const Ssl::CertificateProperties &certProperties)
 {
     debugs(33, 5, "Finding SSL certificate for " << cacheKey << " in cache");
-    Ssl::LocalContextStorage * ssl_ctx_cache = Ssl::TheGlobalContextStorage.getLocalStorage(port->s);
+    Ssl::LocalContextStorage * ssl_ctx_cache = Ssl::TheGlobalContextStorage().getLocalStorage(port->s);
```

Comment:
Here and in other changed lines (where possible):

```suggestion
    const auto ssl_ctx_cache = Ssl::TheGlobalContextStorage().getLocalStorage(port->s);
```

Reply:
Done.